include/libtorrent/alert_types.hpp

```diff
@@ -247,10 +250,18 @@ TORRENT_VERSION_NAMESPACE_2
 #endif
 
 	// The ``torrent_removed_alert`` is posted whenever a torrent is removed. Since
-	// the torrent handle in its base class will always be invalid (since the torrent
+	// the torrent handle in its base class will usually be invalid (since the torrent
 	// is already removed) it has the info hash as a member, to identify it.
 	// It's posted when the ``status_notification`` bit is set in the alert_mask.
 	//
+	// Note that the ``handle`` may stay valid for an arbitrary time after
```

Comment:
I'm not sure about "arbitrary" here. Technically it isn't arbitrary, it's bounded by the session destruction for instance. I think the sentence in the other paragraph where you say this seem better:

"The torrent_handle remains valid for some time after remove_torrent() is called"